R/param-estimates.R

```diff
@@ -31,11 +37,18 @@ param_estimates <- function(.summary) {
 #' @importFrom tibble tibble as_tibble
 #' @importFrom purrr map_at map_depth map_lgl
 #' @importFrom rlang list2
+#' @importFrom stringr str_detect
 #' @export
 param_estimates.bbi_nonmem_summary <- function(.summary) {
   num_methods <- length(.summary[[SUMMARY_PARAM_DATA]])
   param_names <- .summary[[SUMMARY_PARAM_NAMES]]
 
+  # if Bayesian method (includes NUTS) do not return df because it is incorrect and misleading
+  est_method <- .summary[[SUMMARY_DETAILS]][[SUMMARY_EST_METHOD]][[num_methods]]
+  if (str_detect(est_method, "Bayesian")) {
+    stop(glue("param_estimates() is not currently implemented for Bayesian methods. Passed has final estimation method: {est_method}"), call. = FALSE)
```

Comment:
> Passed has final estimation method

What is meant by `Passed`? The summary object? Perhaps we'd consider

> \`.summary\` has final estimation method : {est_method}

Reply:
Oh I think I forgot a noun in there. Good catch. I think I'll just address this in https://github.com/metrumresearchgroup/rbabylon/pull/136 to avoid making a commit with only this change.